laravel/database/migrations/2021_04_13_152249_change_data_type_of_attending_id_column_in_evaluation_data_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class ChangeDataTypeOfAttendingIdColumnInEvaluationDataTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('evaluation_data', function (Blueprint $table) {
+            $table->string('attending_id')->change();
```

Comment:
Note: the attending table's 'id' column is a string because of the way it is parsed from (I think) OSUFindPeople . Instead of changing this column to make it an incrementing id, I've made the foreign key on the evaluation_data table a string.

Reply:
I agree that the types need to match, but I'd prefer we that store integers as integers rather than as strings.

I feel this way because (1) it's confusing and counterintuitive to represent something as a type that it is not and (2) databases are generally more performant when you do things with numbers rather than strings.